tpm2_pytss/ESAPI.py

```diff
@@ -72,7 +72,8 @@ def check_friendly_int(friendly, varname, clazz):
 class ESAPI:
     def __init__(self, tcti=None):
 
-        tctx = ffi.NULL if tcti is None else tcti.ctx
+        self._tcti = tcti
+        tctx = ffi.NULL if tcti is None else tcti.tcti_context
```

Comment:
does a tcti have a non `tcti_context`? wouldn't juct tcti.context be more succinct without losing information?

Reply:
Which tcti do you mean? self._tcti is for (a later) GetTcti method

Reply:
Ahh I misread it, but the `tcti.tcti_context` is the FFI pointer, right? If so, we would want that to be behind a "private" attribute like `tcti._context` or `tcti._ctx`. We want to make it clear that folks shouldn't be using the FFI values by convention.

Unrelated to what you're doing, we'd probably want to make this change in ESAPI as well, I see it's `self.ctx` (sorry my mistake there, ill fix it before release #178.)

Reply:
tcti_context is the FFI pointer, exactly, and it's a property of the TCTI class, self._tcti is later getting the TCTI classed passed to init (so via GetTcti and/or a property of the ESAPI class).

I'll add GetTcti and the tcti property to ESAPI in this PR

Reply:
> tcti_context is the FFI pointer, exactly, and it's a property of the TCTI class, self._tcti is later getting the TCTI classed passed to init (so via GetTcti and/or a property of the ESAPI class).

Can we make the property an underscore property as the FFI types aren't part of the public API.

> 
> I'll add GetTcti and the tcti property to ESAPI in this PR